src/main/java/org/broadinstitute/hellbender/engine/filters/FragmentLengthReadFilter.java

```diff
@@ -9,7 +9,8 @@
 
 /**
  * Keep only read pairs (0x1) with absolute insert length less than or equal to the given value.
- * Taking absolute values allows inclusion of pairs where the considered read's mate is at a smaller genomic coordinate.
+ *
+ * <p>Taking absolute values allows inclusion of pairs where the considered read's mate is at a smaller genomic coordinate.
```

Comment:
where the mate of the read being considered is

Reply:
Done.